dist/upload.js

```diff
@@ -7741,13 +7748,13 @@ async function uploadRecordings({
   metadata
 }) {
   try {
-    const recordingIds = await upload(cli, filter, metadata);
-    const recordings = cli.listAllRecordings({ all: true }).filter((u) => recordingIds.includes(u.recordingId));
+    const recordingIds = await upload(cli, apiKey, filter, metadata);
+    const recordings = cli.listAllRecordings({ all: true }).filter((u) => recordingIds.includes(u.id));
     const uploaded = recordings.filter((u) => u.status === "uploaded");
     const crashed = recordings.filter((u) => u.status === "crashUploaded");
-    console.log("Uploaded", recordingIds.length, "replay(s)");
+    console.log("Uploaded", uploaded.length, "replay(s)");
     console.log("Uploaded", crashed.length, "crash report(s)");
-    if (public2 && recordingIds.length > 0) {
+    if (public2 && uploaded.length > 0) {
```

Comment:
Looks like this prior "fix" wasn't built and so wasn't active. 😬 

Good thing it didn't actually fix things ... I guess?